emscripten.py

```diff
@@ -522,7 +522,9 @@ def compile_settings(compiler_engine, libraries, temp_files):
   # Save settings to a file to work around v8 issue 1579
   with temp_files.get_file('.txt') as settings_file:
     with open(settings_file, 'w') as s:
-      json.dump(shared.Settings.to_dict(), s, sort_keys=True)
+      settings = shared.Settings.to_dict()
+      settings = { key: int(value) if isinstance(value, bool) else value for key, value in settings.items() }
```

Comment:
does `json.dump` not work on `bool` values?

if so maybe it would make sense to have a helper function as this appears in another place too.

Reply:
`True`/`False` in Python is dumped as `true`/`false`. Since I'm making very strict type checking here, only string and integers are accepted, and by intention `true`/`false` is not automatically converted to `0`/`1`. The change is not necessary otherwise.

Reply:
I see, thanks.